ground/gcs/src/plugins/uploader/bl_messages.h

```diff
@@ -95,187 +100,107 @@ enum dfu_partition_label {
     DFU_PARTITION_WAYPOINTS,
     DFU_PARTITION_LOG,
     DFU_PARTITION_OTA,
-};
-
-struct bl_messages {
-    uint8_t flags_command;
+}__attribute__((packed));
```

Comment:
Sorry inadvertently put that back.   I don't have MSVC to compile/test, this was just trying to change it so you can use the PACK() macro you made.